history-of-the-quakers-v2/original/08-book-3-chapter-14.adoc

```diff
@@ -973,15 +1043,20 @@ near twenty years after the establishment of the yearly meeting,
 and exhibits a specimen of their watchful care against everything that
 might dishonour or defile the reputation of their society.
 
-"`Advised that none launch into trading and worldly business
+[.embedded-content-document]
```

Comment:
A minute also?

Reply:
not sure.  could be minute, or epistle.  but i wonder if we should add a type "advice".  But... I guess let's just leave it untyped, since it's unclear.